pkg/virt-controller/watch/vmi.go

```diff
@@ -1056,6 +1086,21 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 	}
 
 	if !isTempPod(pod) && isPodReady(pod) {
+		if vmispec.SriovInterfaceExist(vmi.Spec.Domain.Devices.Interfaces) {
+			networkPciMapAnnotationValue, err := sriov.CreateNetworkPCIAnnotationValue(
+				vmi.Spec.Networks, vmi.Spec.Domain.Devices.Interfaces, pod.Annotations[networkv1.NetworkStatusAnnot],
+			)
+			if err != nil {
+				log.Log.V(3).Object(vmi).Warningf("failed to prepare SR-IOV network-pci map %q", err)
```

Comment:
Does it make sense to make the user aware of it (SR-IOV mapping might be wrong)?
If so, it should be raised as a warning event instead of a log message.

Reply:
removed the V(3)